### DISCUSSION TITLE
gold db: postgres db or scema

### DISCUSSION THREAD

**Discussion by josephyys** [zsvoboda/ngods-stocks#7] [Q&A] [answered]
Hi,

At first thanks for the awesome work!

I have test it but fail at dagster process at task: dbt_gold_test_doc_sources_op,
It seems that there is missing postgres data in ./data/postgres?

Joseph

**Comment by zsvoboda** [comment, marked as answer]
Hi Joseph, 

thank you! The ./data/postgres directory is mapped to the Postgres database directory in the postgres container (see the docker-compose.yml. The directory should be created and populated with the Postgres DB when the Postgres container starts. Can you please share the command line that you use for running the docker-compose?

Regards,

ZD

**Reply by josephyys** [reply]
Hi ZD,

Thanks for reply.

I use "docker-compose -f docker-compose.x86.yml up -d" because I want to use DataHub too, 
and I try to recreate postgres:
 docker-compose -f docker-compose.x86.yml  create  postgres --force-recreate --build 
the gold database is autocreated now.

I think maybe it's related with some problems I encountered previously in Windows:
1. Because firewall in my office environment, I add dns setting in docker-compose.x86.yml:
    dns:
      - 8.8.8.8
      - 4.4.4.4

2. The enteypoint.sh, notebook  in aio is auto converted to Windows CRLF format,  
I changed the format to LF and restart aio  successfully by: 
docker-compose -f docker-compose.x86.yml  create  aio --force-recreate --build 


Thanks for your help!

Regards,
Joseph
